app/templates/interfaceactiontable.html

```diff
@@ -30,6 +30,7 @@ <h1>{{ page_title_value }}</h1>
                             'distinct_job_roles': distinct_job_roles,
                             'previous_ons_id': previous_ons_id,
                             'previous_select': previous_select,
+			    'previous_employee_id': previous_employee_id,
```

Comment:
The indentation here seems to be off! It may have inserted tabs instead of spaces.